plugins/snowflake/setup.py

```diff
@@ -48,6 +48,7 @@
     install_requires=[
         'dbt-core=={}'.format(package_version),
         'snowflake-connector-python==2.2.10',
+        'snowflake-connector-python[secure-local-storage]',
```

Comment:
I think the line above can just read
```
snowflake-connector-python[secure-local-storage]==2.2.10
```

I think these two things might be equivalent, but it might make sense to combine them here